Dockerfile

```diff
@@ -0,0 +1,55 @@
+FROM python:2.7.12
+
+ARG BUILD_DEPS=" \
+    build-essential \
+    git \
+    libffi-dev \
+    libssl-dev \
+    libxml2-dev \
+    libxslt1-dev"
+
+RUN apt-get update && \
+    apt-get install \
+        --no-install-suggests \
+        --no-install-recommends \
+        -y \
+        $BUILD_DEPS && \
+    rm -rf /var/lib/apt/lists/*
+
+# Node
+RUN curl -sL https://deb.nodesource.com/setup_6.x | bash -
+RUN apt-get install -y nodejs
+
+RUN mkdir /code
+RUN mkdir /code/website
+RUN mkdir /code/plugins
+WORKDIR /code/website
+
+# Python dependencies
+RUN pip install uWSGI==2.0.13.1
+COPY ./requirements.txt /code/website
+RUN pip install -r requirements.txt
+
+# Node dependencies
+COPY ./package.json /code/website
+RUN npm install
+
+COPY . /code/website
+
+# Translations
+RUN pybabel compile -d website/frontend/translations
+
+# Static files
+RUN ./node_modules/.bin/gulp
+
+# Plugins
+RUN git clone https://github.com/metabrainz/picard-plugins.git /code/plugins
+WORKDIR /code/plugins
+RUN python generate.py
```

Comment:
Would `python generate.py /code/plugins/generate.py` work instead of setting the WORKDIR? Only asking since it's annoying when I sh into a container and end up in a weird directory. :)